pkg/rancher/migrations.go

```diff
@@ -430,3 +415,40 @@ func migrateCAPIMachineLabelsAndAnnotationsToPlanSecret(w *wrangler.Context) err
 	cm.Data[capiMigratedKey] = "true"
 	return createOrUpdateConfigMap(w.Core.ConfigMap(), cm)
 }
+
+func insertOrUpdateCondition(d data.Object, desiredCondition summary.Condition) (bool, error) {
```

Comment:
This is from the machine-provision controller. "A little copying is better than a little dependency."